dist/hawkular-alerts-openshift.yml

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright 2015-2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+id: hawkular-alerts
+kind: Template
+apiVersion: v1
+name: Hawkular Alerts
+metadata:
+  name: hawkular-alerts
+
+
+objects:
+- apiVersion: extensions/v1beta1
```

Comment:
This "v1beta1" refers to the API version of OpenShift or Alerting ?

Reply:
that is a specific OpenShift (actually it might be K8S) version. Has nothing to do with alerts.